common/test/Configuration.IoTHub.cs

```diff
@@ -16,6 +16,8 @@ public static partial class IoTHub
 
             public static X509Certificate2 GetCertificateWithPrivateKey() 
                 => GetBase64EncodedCertificate("IOTHUB_X509_PFX_CERTIFICATE");
+
+            public static string EventHubString => GetValue("EVENTHUB_CONN_STRING_CSHARP", string.Empty);
```

Comment:
>string.Empty [](start = 91, length = 12)

do we want to use the default value of null for the string?

Reply:
I chose to use string.Empty  because using null wasn't working 100% of the time but string.Empty was. It has the same desired effect.

Reply:
what do you mean by not 100% time?  do you mean GetBase64EncodedCertificate("IOTHUB_X509_PFX_CERTIFICATE"); might give you different results on different runs?  If yes, we might have found the fix for the  e2e tests failing randomly when retrieving x509 certs?

Reply:
Null is the default functionality on GetVaule. GetBase64encoded takes a default value, but we don't pass it into GetValue, which I have just changed. I just tested it with string.Empty and it didn't fix the issue

Reply:
Whatever we choose here, when the tests fail because this env not being set, the error should be very easy to understand.